### PR TITLE
docs(aiplatform): Add MultimodalMultiImage.cs #2462

### PR DESCRIPTION
**Pull request by meteatamel**

### CODE REVIEW

File: aiplatform/api/AIPlatform.Samples.Tests/GeminiMultimodalMultiImageTest.cs [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2023 Google LLC
```
Comment (amanda-tarafa) [inline aiplatform/api/AIPlatform.Samples.Tests/GeminiMultimodalMultiImageTest.cs:2]: nit 2024

File: aiplatform/api/AIPlatform.Samples/GeminiMultimodalMultiImage.cs [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                {
+                    MimeType = "image/png",
+                    Data = christRedeemer
+                }
+            }
+        });
+
+        var generateContentRequest = new GenerateContentRequest
+        {
+            Model = $"projects/{projectId}/locations/{location}/publishers/{publisher}/models/{model}"
```
Comment (amanda-tarafa) [inline aiplatform/api/AIPlatform.Samples/GeminiMultimodalMultiImage.cs:96]: Same question as in #2461 about the resource name helper.

File: aiplatform/api/AIPlatform.Samples/GeminiMultimodalMultiImage.cs [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+        generateContentRequest.Contents.Add(content);
+
+        // Make the request, returning a streaming response
+        using PredictionServiceClient.StreamGenerateContentStream response = predictionServiceClient.StreamGenerateContent(generateContentRequest);
+
+        StringBuilder fullText = new();
+
+        // Read streaming responses from server until complete
+        AsyncResponseStream<GenerateContentResponse> responseStream = response.GetResponseStream();
+        while (await responseStream.MoveNextAsync())
```
Comment (amanda-tarafa) [inline aiplatform/api/AIPlatform.Samples/GeminiMultimodalMultiImage.cs:107]: `await foreach` maybe?

**Review by amanda-tarafa — COMMENTED** [review]
Same comments as for #2461

File: aiplatform/api/AIPlatform.Samples/GeminiMultimodalMultiImage.cs (hunk 2 above)
Comment (meteatamel) [reply aiplatform/api/AIPlatform.Samples/GeminiMultimodalMultiImage.cs]: No such method unfortunately

File: aiplatform/api/AIPlatform.Samples/GeminiMultimodalMultiImage.cs (hunk 3 above)
Comment (meteatamel) [reply aiplatform/api/AIPlatform.Samples/GeminiMultimodalMultiImage.cs]: Fixed

**Review by amanda-tarafa — APPROVED** [review]

**Pull request merged**